docs/06_tutorial3.md

```diff
@@ -165,197 +203,249 @@ make install
-    <source src="/img/tutorials/meshlab-segment4.mp4" type="video/mp4"/>
-  </video>
-</figure>
+* /layers/ contains files numbered 00.tif to 64.tif (or 000.tif to 156.tif for 3.24um scans). These are slices of the surface volume, each of a thickness equal to the voxel size of the scroll you are currently segmenting, typically what we call “low res” are the 7.91um voxel spacing, and “high res” would be 3.24um voxel spacing. For a 7.91um volume, this means that each surface volume is 64 tifs * 7.91um, for about 506um thickness, or .506mm
 
-The mesh uses the .tif file as a texture, so you can actually see the 3D placement of the metallic ink.
+* `<segmentationnumber>.obj` this is a 3D image format that stores information about the geometry of a 3D model, such as vertex positions, texture maps, normals, and faces. This is a mesh created from the pointset created by your segmentation line and detailed further below
 
-If you additionally load the full mesh we created at the end of [“Tutorial 2”](tutorial2#meshes), you can more clearly see the location of this segment with respect to the rest of the scroll. For this to work, you might need to generate a full mesh again using Fiji (just like in ["Tutorial 2: Scanning"](tutorial2#meshes)), but this time using the image stack in the `campfire.volpkg/volumes` directory, since `vc_packager` might have done some transformations to the slices.
+* `<segmentationnumber>.tif` is a composite image of maximum intensity pixels from within the surface volume, it is also used for texturing in combination with the mtl file
 
-<figure>
-  <video autoPlay playsInline loop muted className="max-w-[100%] rounded-xl" poster="/img/tutorials/meshlab-full4.jpg">
-    <source src="/img/tutorials/meshlab-full4.webm" type="video/webm"/>
-    <source src="/img/tutorials/meshlab-full4.mp4" type="video/mp4"/>
-  </video>
-</figure>
+* `<segmentationnumber>.mtl` is a text file that contains information about the material, such as opacity, reflectivity, and points to the texture file (this tif in the previous line)
+
+* `<segmentationnumber>.ppm` A simple way to think about this is to think of it as a super dense point cloud that for each voxel in the flattened volume contains a point representing the 3d location in the original volume that voxel came from, and some information about that point. A better explanation is that a .ppm is special case of an ordered pointset, where type = double and dim = 6. It is generated by flattening the segmented surface, discretizing the parameterization, and storing the corresponding 3D point and surface normal for each spot on the surface. Using slice notation, ppm[y, x, 0:3]  stores the 3D point and ppm[y, x, 3:6] stores the surface normal. Because the surface does not fill the 2D array entirely, there is a *_mask.png file next to the PPM. It stores which pixels in the PPM have valid values. More info here:  Volume Cartographer PPM file format (github.com)
```

Comment:
```suggestion
* `<segmentationnumber>.ppm` A simple way to think about this is to think of it as a super dense point cloud that for each voxel in the flattened volume contains a point representing the 3d location in the original volume that voxel came from, and some information about that point. A better explanation is that a .ppm is special case of an ordered pointset, where type = double and dim = 6. It is generated by flattening the segmented surface, discretizing the parameterization, and storing the corresponding 3D point and surface normal for each spot on the surface. Using slice notation, ppm[y, x, 0:3] stores the 3D point and ppm[y, x, 3:6] stores the surface normal. Because the surface does not fill the 2D array entirely, there is a *_mask.png file next to the PPM. It stores which pixels in the PPM have valid values. More info here: Volume Cartographer PPM file format (github.com)
```
Removes redundant spacing.